packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/TypeProvider.cs

```diff
@@ -424,7 +424,7 @@ private static bool IsMatch(TypeProvider enclosingType, MethodSignatureBase sign
             }
             else if (attribute.ConstructorArguments[1].Kind != TypedConstantKind.Array)
             {
-                parameterTypes = [(ISymbol?) attribute.ConstructorArguments[1].Value];
```

Comment:
This was incorrectly only using the first value when there were multiple parameters and the 1st parameter was not an array.